src/app/services/signalr/signalr.service.ts

```diff
@@ -73,6 +82,7 @@ export class SignalRService {
     if (this.userId && this.viewId) {
       this.joinUser(this.userId, this.viewId);
     }
+
```

Comment:
Add the following here so that the client rejoins the vm group if their connection drops and reconnects.

```
if (this.vmId) {
  this.setActiveVirtualMachine(vmId)
}
```

Reply:
Correct, I'm removing this.vmId.  That was necessary for an older version I had working.